package.json

```diff
@@ -153,7 +153,7 @@
     "@ngrx/store-devtools": "^14.3.0",
```

Comment:
Overall will we upgrade the minimum version to 14.2?  Is there a point to leave it at 14.1.3 ?

Reply:
We agreed to do this in a follow up ticket if we go forward with this.